Assignment3/modules.rb

```diff
@@ -0,0 +1,57 @@
+#Module File for Assignment3
+require_relative 'data'
+#Input Module for Country Name
+module GetCountry
+  def self.GetCname
+    puts "Enter Country Name"
+    cname = gets.chomp.downcase
+    return  cname
+  end
+end
+#Module to find Country name in Data
+module FindCountry
+  def self.FindCname
+    cname = GetCountry.GetCname
+    count = CountryData.Country.find {|x| x[:Name] == cname}
+    if count != nil
```

Comment:
You can do this with ternary operator as:
count.present? ? return count : puts "Country Not Found in our Database"